test/LinkerSections/function_sections_ir_check.swift

```diff
@@ -0,0 +1,12 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-build-swift -Xfrontend -function-sections -emit-library -emit-ir -static -parse-stdlib %S/Inputs/FunctionSections.swift | %FileCheck %s
+
+// CHECK: define protected swiftcc void @"$s16FunctionSections5func1yyF"() #0 {
+// CHECK: entry:
+// CHECK:   ret void
+// CHECK: }
+
+// CHECK: define protected swiftcc void @"$s16FunctionSections5func2yyF"() #0 {
```

Comment:
These are going to fail to match on Windows

Reply:
Different mangling? Should we match with a regex instead then?

Reply:
Done, I think.